HLTrigger/Configuration/python/Tools/confdbOfflineConverter.py

```diff
@@ -27,8 +27,8 @@ def CheckTempDirectory(dir):
 
     def __init__(self, database = 'hltdev', url = None, verbose = False):
         self.verbose = verbose
```

Comment:
this is a temporary workaround to use the modified Java converter, until it is fully validated and deployed